trestle/transforms/utils/osco_helper.py

```diff
@@ -63,6 +63,7 @@ class RuleUse():
 
     def __init__(
```

Comment:
For these classes we can use Pydantic base_models that will automate some of this setup.

Reply:
We can leave this for now though. Just a thought to decrease total LoC